depends/packages/libpng.mk

```diff
@@ -0,0 +1,23 @@
+package=libpng
+$(package)_version=1.6.34
```

Comment:
If we are going to add this to depends, we should at least add the most recent release ([1.6.37](http://www.libpng.org/pub/png/libpng.html)); unless there's some reason not to, in which case, we should document why.